src/MinimalDelegation.sol

```diff
@@ -191,18 +193,41 @@ contract MinimalDelegation is
     }
 
     /// @inheritdoc ERC1271
+    /// @dev WrappedSignature is used for both NestedTypedDataSign and NestedPersonalSign signatures.
+    ///      If the caller implementing ERC-1271 is considered safe, we verify the signature over the data directly
+    /// TypedDataSign signatures are of the form: abi.encode(bytes32, bytes(`signature ‖ APP_DOMAIN_SEPARATOR ‖ contentsHash ‖ contentsDescr ‖ uint16(contentsDescr.length)`))
```

Comment:
```suggestion
    /// TypedDataSign signatures are of the form: abi.encode(bytes32, bytes(`signature ‖ APP_DOMAIN_SEPARATOR ‖ contentsHash ‖ contentsDescr ‖ uint16(contentsDescr.length)`), bytes)
```

Reply:
I think missing the bytes hookData encoding?

Reply:
Ah wait I see, maybe we need two natspec comments here 
1- on the outermost wrapped sig
2- on the innermost signature if it is a 7339 TypedDataSign compliant